aunt-leahs-app/src/components/adminLocationList/locationList.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { List, Fab } from '@material-ui/core';
+import { LocationListItem } from './locationListItem.jsx';
+import AddIcon from '@material-ui/icons/Add';
+import CustomButton from '../customButton'
+
+export class LocationList extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            locations: [
+                {
+                    name: "Tree Lot",
+                    id: 1,
+                    address: "1234 East Mall"
+                },
+                {
+                    name: "Thrift Store",
+                    id: 2,
+                    address: "1234 Robson St"
+                },
+                {
+                    name: "Fundraising Event",
+                    id: 3,
+                    address: "1234 Wesbrook Mall"
+                }],
+            newLocations: [],
```

Comment:
Should locations and new locations be two separate objects in the state? I might have overlooked something but is there a reason we can't push/pop on locations directly?

Also, this is more on the design end than the actual code, but how do you add the address of a new location?

Reply:
They need to be 2 separate arrays because we don't have a unique id for a new location until we add something to the database. So basically, we need this because we allow the user to do all kinds of changes before "Save"ing to the database. We discussed this extensively before the break and this was the solution that the team wanted to use.

Reply:
Right, sorry! Must have missed this conversation. Looks great otherwise!